Gordon360/Controllers/ProfilesController.cs

```diff
@@ -611,5 +629,20 @@ public ActionResult<IEnumerable<MembershipHistoryViewModel>> GetMembershipHistor
 
             return Ok(membershipHistories);
         }
+
+        /// <summary>
+        /// Return a list of mail destinations' descriptions.
+        /// </summary>
+        /// <returns> All Mail Destinations</returns>
+        [HttpGet]
+        [Route("mail_stops")]
+        public ActionResult<IEnumerable<string>> GetMailStops()
+        {
+            var involvements = _webSQLContext.Mailstops.Select(m => m.code)
```

Comment:
```suggestion
            var = _webSQLContext.Mailstops.Select(m => m.code)
```
I don't think involvements is the right variable name here